src/client/src/commons/form/profile/components/profileLayers/components/lithologicalDescriptionLayers/actionButtons.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Stack, Tooltip } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import ClearIcon from "@mui/icons-material/Clear";
+import ModeEditIcon from "@mui/icons-material/ModeEdit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+
+const ActionButtons = props => {
+  const {
+    item,
+    changeSelectionAndSubmit,
+    addMutation,
+    selectedDescription,
+    selectedStratigraphyID,
+    setDescriptionIdSelectedForDelete,
+  } = props;
+  const { t } = useTranslation();
+
+  return (
+    <Stack direction="row" sx={{ marginLeft: "auto", padding: "3px" }}>
+      {selectedDescription?.id !== item?.id && item.id !== null ? (
+        <>
+          <Tooltip title={t("edit")}>
+            <ModeEditIcon onClick={() => changeSelectionAndSubmit(item)} />
+          </Tooltip>
+          <Tooltip title={t("delete")}>
+            <DeleteIcon
+              sx={{ color: "red", opacity: 0.7 }}
+              onClick={() => {
+                setDescriptionIdSelectedForDelete(item.id);
+              }}
+            />
+          </Tooltip>
+        </>
+      ) : (
+        item.id === null && (
+          <Tooltip title={t("add")}>
+            <AddCircleIcon
+              style={{ color: "#9f3a38" }}
+              onClick={() => {
+                addMutation.mutate({
+                  stratigraphyId: selectedStratigraphyID,
+                  fromDepth: item.fromDepth,
+                  toDepth: item.toDepth,
+                });
+              }}
+            />
+          </Tooltip>
+        )
+      )}
+      {selectedDescription?.id === item?.id && (
+        <Tooltip title={t("stop-editing")}>
+          <ClearIcon onClick={() => changeSelectionAndSubmit(null)} />
+        </Tooltip>
+      )}
+    </Stack>
```

Comment:
PP: Die Buttons haben herzlich wenig miteinander zu tun, da sie je nach state nicht miteinander sichtbar sind. Ich würde sie daher nicht in einer Componente zusammenfassen.